lib/containers/common.pm

```diff
@@ -247,4 +247,19 @@ sub test_rpm_db_backend {
     }
 }
 
+sub test_systemd_install {
+    my %args = @_;
+    my $image = $args{image};
+    my $runtime = $args{runtime};
+
+    die 'Argument $image not provided!' unless $image;
+    die 'Argument $runtime not provided!' unless $runtime;
+
+    my ($running_version, $sp, $host_distri) = get_os_release("$runtime run $image");
+    # TW and SLE 15-SP3+ uses rpm-ndb in the image
```

Comment:
Wrong comment?

Reply:
Yup, good catch! copy and paste from the rpm-db function.